### DISCUSSION TITLE
8.15买的1年专业版，到现在还是免费版，已经发过邮件了一星期了还没回应，再不处理我就从渠道退款了

### DISCUSSION THREAD

**Discussion by xiaopangusers123456** [getlantern/lantern-forum-cn#843] [General]
邮箱 xtjxyc@163.com  
要么给我激活码   要么因为收不到验证码的原因给我转邮箱 xtjxyc@gmail.com  ，要么退款
![微信图片_20240830223734](https://github.com/user-attachments/assets/fb2ae449-ad6e-4f55-860f-e3861e123aac)

**Comment by xiaopangusers123456** [comment]
又是4天过去了，装死了吗

**Comment by xiaopangusers123456** [comment]
已经1个多月了，这么个论坛就这么点帖子，我翻到第二页就翻到我上周发的了
我理解国内163邮箱接收不到验证码 但是我也需要合理的解决方法
要么把我的邮箱改为xtjxyc@gmail.com ，我可以自己去收到授权码去登录专业版
要么直接给我激活码

总不能装死吧

**Comment by xiaopangusers123456** [comment]
已经1个多月了，这么个论坛就这么点帖子，我翻到第二页就翻到我上周发的了
我理解国内163邮箱接收不到验证码 但是我也需要合理的解决方法
要么把我的邮箱改为xtjxyc@gmail.com ，我可以自己去收到授权码去登录专业版
要么直接给我激活码

总不能装死吧

**Comment by MaxMKMax** [comment]
@xiaopangusers123456 不好意思. 不知道为什么一直没看到您的帖子. 不是不回复你. 非常抱歉!

**Comment by MaxMKMax** [comment]
@xiaopangusers123456 马上给您提交问题上去. 尽快给您解决问题. 给我们几天时间. 支付部门的人不在线! 万分抱歉啊!

**Comment by greySkiesMarsh** [comment]
@xiaopangusers123456 

Hello, I've fixed the problem with your account in our system. Please try to login again, and if you're still unable to receive the link code in your email, then I'll be happy to change your account to use the gmail address you've posted.

**Comment by MaxMKMax** [comment]
@xiaopangusers123456 问题应该已经解决了. 谢谢您支持蓝灯!
